api/tenure.js

```diff
@@ -13,7 +12,7 @@ export const getTenure = (tenureId) => {
         cy.request({
             method: 'GET',
             url: `${tenureEndpoint}/tenures/${tenureId}`,
-            headers: { Authorization: `Bearer ${envConfig.gssoTestKey}` }
+            headers: { Authorization: `Bearer ${Cypress.config("gssoTestKey")}` }
```

Comment:
It's repeated everywhere. Extract into var?

Reply:
Do you mean something like a global variable?

Reply:
yh, I'm guessing it's the same every time.
So may as well make it global within the file scope to only make it compute once.

Reply:
Seems like there isn't a simple way to set a global variable other than cypress config or using [environment variables](https://stackoverflow.com/questions/49700479/cypress-how-to-store-global-constants-in-a-file-that-can-be-used-across-all-spec).